marketplace/optimizely/test/AppPage.spec.js

```diff
@@ -52,13 +38,39 @@ describe('AppPage', () => {
 
     const sdk = { ...basicProps.sdk, platformAlpha, space };
 
-    const props = { ...basicProps, accessToken: '123', client, sdk };
+    const props = { ...basicProps, accessToken: '123', sdk };
 
-    const { container } = render(<AppPage {...props} />);
+    const { container, getByTestId } = render(<AppPage {...props} />);
+
+    await wait(() => {
+      if (!configFunc) {
+        throw '';
+      }
+    });
 
-    await wait();
     await configFunc();
-    await wait();
+    expect(container).toMatchSnapshot();
+
+    act(() => {
+      fireEvent.click(getByTestId('add-content'));
+    });
+
+    getByTestId('content-type-selector').click();
+
+    getByTestId('content-type-selector').firstChild.click();
```

Comment:
why do we need to click on parent first and then on its child?

Reply:
This is a select element. I was hoping to mimic clicking on the select itself, then clicking on the first `option`.

Maybe I should use more testIds on the `option`s as well...

Reply:
ah, I see... just a comment would help here.

Reply:
Done in: b686dddba2d5fce7dab7aa98df8e5eff34002461